src/js/apps/globals/nav_app.js

```diff
@@ -159,6 +160,10 @@ export default App.extend({
     navView.showChildView('patients', patientsCollectionView);
     navView.showChildView('worklists', workflowsCollectionView);
 
+    this.listenTo(navView, 'search', () => {
+      this.onSearch(navView);
```

Comment:
I think maybe this should be `showSearchModal`.  It seems awkward to call a `on___` method directly.